cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -244,8 +247,8 @@ spec:
 * Create an ASG with a MixedInstancesPolicy that refers to the newly-created LT.
 * Set LaunchTemplateOverrides to include the 'base' instance type r5.2xlarge and suitable alternatives, e.g. r5d.2xlarge, i3.2xlarge, r5a.2xlarge and r5ad.2xlarge. Differing processor types and speeds should be evaluated depending on your use-case(s).
 * Set [InstancesDistribution](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_InstancesDistribution.html) according to your needs.
-* See [Allocation Strategies](https://docs.aws.amazon.com/autoscaling/ec2/userguide/asg-purchase-options.html#asg-allocation-strategies) for information about the ASG fulfils capacity from the specified instance types.
-* Repeat by creating other LTs and ASGs, for example c5.18xlarge and c5n.18xlarge or a bunch of similar burstable instances.
+* See [Allocation Strategies](https://docs.aws.amazon.com/autoscaling/ec2/userguide/asg-purchase-options.html#asg-allocation-strategies) for information about the ASG fulfils capacity from the specified instance types. It is recommended to use the capacity-optimized allocation strategy, which will automatically launche Spot Instances into the most available pools by looking at real-time capacity data and predicting which are the most available
+* For the same workload or for the generic capacity in your cluster, you can also create more node groups with identical number of vCPU/Mem, but from different sizes. For example, node group 1: m5.xlarge, m5a.xlarge, m5d.xlarge, m5ad.xlarge, m4.xlarge. node group 2: m5.2xlarge, m5a.2xlarge, m5d.2xlarge, m5ad.2xlarge, m4.2xlarge. This approach along with cluster-autoscaler will help you tap into many Spot capacity pools, and increase your chances of achieving your desired scale.
```

Comment:
```identical number of vCPU/Mem```

Do you mean the ratio of vCPU/Mem here?

Reply:
What I meant was that each node group should have instance types that have the same number of vCPUs/Mem, so it's not really a ratio thing right? does the full sentence not explain this approach properly? appreciate any suggestions

Reply:
OK, I understand now, I think the use of `vCPUs/Mem` and the fact the second node group was all instances double the size of the first threw me into looking at it as a ratio thing of providing different node groups but with the same ratio of vCPUs to Memory. Maybe clearer if it's `identical numbers of vCPUs and Memory`?

Reply:
OK, made it a bit clearer. WDYT?